src/util/utils.ts

```diff
@@ -23,19 +23,46 @@ import {
   SyntaxKind
 } from 'typescript';
 import { getDeclaredMethods } from './classDeclarationUtils';
+import { objectKeys } from './objectKeys';
 
-export enum Decorators {
+export enum AngularClassDecorators {
+  Component = 'Component',
+  Directive = 'Directive',
+  Injectable = 'Injectable',
+  NgModule = 'NgModule',
+  Pipe = 'Pipe'
+}
+
+enum AngularMethodDecorators {
+  HostListener = 'HostListener'
+}
+
+enum AngularParameterPropertyDecorators {
```

Comment:
Hmm, I've opted to call it `AngularParameterPropertyDecorators` because typescript itself calls constructor parameters as `parameter properties`.

I've changed it to `AngularConstructorParameterDecorators`. Does it sounds better for you?